### PR TITLE
Fixes #25636: New README for Rudder

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/25636

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: README.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+Rudder has adopted an _open-core_ model:
+
+* **Rudder Core** is available for free and totally open-source (under GPL 3.0 an Apache 2.0). It includes the main Rudder components, Linux agents, plus several plugins.
+* The complete **Rudder** solution is paid and partially open-source. A part of the paid plugins and agents are distributed under proprietary license. It also includes other benefits:
+
+- Additional plugins that add major features
+- Support for older versions on Linux distributions
+- Support for additional architectures (ARM & Power)
+- Support for proprietary operating systems: Windows and AIX
```
Comment (fanf) [inline README.md:117]: I don't think we want to advertise AIX here.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+
+### Is Rudder open-source? Is it free? What is Rudder Core?
+
+Rudder has adopted an _open-core_ model:
+
+* **Rudder Core** is available for free and totally open-source (under GPL 3.0 an Apache 2.0). It includes the main Rudder components, Linux agents, plus several plugins.
+* The complete **Rudder** solution is paid and partially open-source. A part of the paid plugins and agents are distributed under proprietary license. It also includes other benefits:
+
+- Additional plugins that add major features
```
Comment (fanf) [inline README.md:114]: having a shift for the second list would make it clearer (with 4 spaces before -)

File: README.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+The full Rudder solution is a direct super-set of Rudder Core, so you don't have to
+reinstall anything and your server and agents are compatible with the additional
+features which come as plugins.
+
+You may just need to change the repository URL on your systems to benefit from the extended
+maintenance in the future.
+
+### What is the governance structure of the Rudder project?
+
+The project is lead by the French (🇫🇷🇪🇺) company of the same name (previously known as *Normation*).
```
Comment (fanf) [inline README.md:140]: "French laws apply to the project."
Or something to give the legal juridiction

Comment (fanf) [reply README.md:140]: (it's implied by french company, perhaps it's enough actually)

File: README.md (hunk 1 above)
Comment (amousset) [reply README.md:117]: indeed!

File: README.md (hunk 3 above)
Comment (amousset) [reply README.md:140]: I think the country is enough, especially in a readme.

Speaking about legal jusrisdiction looks a bit threatening to me.

**Comment by amousset** [comment]
PR updated with a new commit

**Review by fanf — APPROVED** [review]
Good to see that landing!

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**